src/fides/api/alembic/migrations/versions/32497f08e227_translation_cleanup.py

```diff
@@ -0,0 +1,380 @@
+"""translation_cleanup Cleans up deprecated tables and fields after multitranslation migration
+
+Revision ID: 32497f08e227
+Revises: 0c65325843bd
+Create Date: 2024-03-01 22:02:51.248873
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "32497f08e227"
+down_revision = "0c65325843bd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column("privacypreferencehistory", "privacy_experience_id")
+    op.drop_column("servednoticehistory", "privacy_experience_id")
+
+    op.drop_index(
+        "ix_privacyexperienceconfig_banner_enabled",
+        table_name="privacyexperienceconfig",
+    )
+    op.drop_column("privacyexperienceconfig", "description")
+    op.drop_column("privacyexperienceconfig", "banner_enabled")
+    op.drop_column("privacyexperienceconfig", "title")
+    op.drop_column("privacyexperienceconfig", "is_default")
+    op.drop_column("privacyexperienceconfig", "accept_button_label")
+    op.drop_column("privacyexperienceconfig", "version")
+    op.drop_column("privacyexperienceconfig", "reject_button_label")
+    op.drop_column("privacyexperienceconfig", "banner_title")
+    op.drop_column("privacyexperienceconfig", "privacy_preferences_link_label")
+    op.drop_column("privacyexperienceconfig", "privacy_policy_link_label")
+    op.drop_column("privacyexperienceconfig", "privacy_policy_url")
+    op.drop_column("privacyexperienceconfig", "acknowledge_button_label")
+    op.drop_column("privacyexperienceconfig", "banner_description")
+    op.drop_column("privacyexperienceconfig", "save_button_label")
+    op.drop_index(
+        "ix_privacyexperienceconfighistory_banner_enabled",
+        table_name="privacyexperienceconfighistory",
+    )
+    op.drop_column("privacyexperienceconfighistory", "banner_enabled")
```

Comment:
Same here this historical concept might need to be retained for auditing purposes - instead setting these to nullable and not collecting them going forward, but leaving old data. Will check with product.